### PR TITLE
Feature create icon component

### PR DESCRIPTION
**Pull request by daoanhhuy26012001**
- #49 
- [x] Install and import module @fortawesome.
- [x] Create element <FontAwesomeIcon>
- [x] Build & test elements on storybook
- **NOTE**:
  - The three component objects I create in the storybook will correspond to: image, image, image
  - See more details: [Link](https://docs.google.com/document/d/1iTrT9cdKrMCjUc6aDLS514gEk7BUFj-m2SCs2wCy2Lw/edit#heading=h.7t5pbcmau6ih)
- **RESULT**:
image

### CODE REVIEW

File: ReactJS/practice-one/src/components/Common/Icon/Icon.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import down from '../../../assets/images/down.png';
```
Comment (phuocduong-agilityio) [inline ReactJS/practice-one/src/components/Common/Icon/Icon.stories.jsx:1]: Could you collect it as constant?

File: ReactJS/practice-one/src/components/Common/Icon/icon.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.icon {
+  cursor: pointer;
+}
+
+.icon .icon--photo {
```
Comment (phuocduong-agilityio) [inline ReactJS/practice-one/src/components/Common/Icon/icon.css:5]: --xxx for modifier, but this is not. Pls double check

Comment (daoanhhuy26012001) [reply ReactJS/practice-one/src/components/Common/Icon/icon.css:5]: Done

File: ReactJS/practice-one/src/components/Common/Icon/Icon.stories.jsx (hunk 1 above)
Comment (daoanhhuy26012001) [reply ReactJS/practice-one/src/components/Common/Icon/Icon.stories.jsx]: Done

**Pull request merged**